inference/core/interfaces/webrtc_worker/__init__.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+import multiprocessing
+
+from inference.core.env import WEBRTC_MODAL_TOKEN_ID, WEBRTC_MODAL_TOKEN_SECRET
+from inference.core.interfaces.stream_manager.manager_app.entities import (
+    InitialiseWebRTCPipelinePayload,
+)
+
+from .cpu import rtc_peer_connection_process
+from .modal import spawn_rtc_peer_connection_modal
```

Comment:
seems like this would fail if modal isn't setup since the function is defined inside if block in that module